lib/axp209.js

```diff
@@ -15,12 +15,16 @@ function AXP209(bus, address) {
   this._i2c = new I2C(bus, address);
 
   this._reads = {};
+  
 }
 
 util.inherits(AXP209, events.EventEmitter);
 
 AXP209.prototype.open = function() {
   this._i2c.open();
+  
+  this._configureVcc18Adc();
+  //this._resetVcc18();
```

Comment:
Hm good idea.. would it be something like this?
```
  if (pin === 'VCC-1V8' && mode === 'ANALOG') {
      this._configureBatAdc();
  }
```

Because we only want to call it if they are using that specific pin as analog, not just any pin as analog